andela/travel_tool_front
# Implement requests table(UI only)

#### What does this PR do?
Impement the table for requests

#### Description of Task to be completed?
- Create table for requests
- Create pagination component
- Style created components
- Test all created components
- Create route for requests page
- Set up sinon and enzyme for tests
- Link line-awesome cdn for icons

#### How should this be manually tested?
- Clone the repo
- Checkout to this branch - git checkout ch-requests-table-159392486
- Run ```./docker.sh development``` to set up and start the app
- Navigate to `localhost:3000/requests` in the browser and you should see the table containing requests.

#### Any background context you want to provide?
This feature will have to be integrated with other components in the requests page

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[159392486](https://www.pivotaltracker.com/story/show/159392486)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-08-06 at 7 55 12 pm" src="https://user-images.githubusercontent.com/25195857/43735554-c89ea170-99b2-11e8-9061-75b9420ec8d3.png">
<img width="1440" alt="screen shot 2018-08-06 at 7 58 11 pm" src="https://user-images.githubusercontent.com/25195857/43735673-1fdff650-99b3-11e8-8f46-3980a9b234cf.png">


#### Questions:
N/A
